homeassistant/components/zwave_js/strings.json

```diff
@@ -385,6 +385,44 @@
           "description": "The Notification Event number as defined in the Z-Wave specs."
         }
       }
+    },
+    "set_lock_configuration": {
+      "name": "Set lock configuration",
+      "description": "Sets the configuration for a lock.",
+      "fields": {
+        "operation_type": {
+          "name": "Operation Type",
+          "description": "The operation type of the lock."
+        },
+        "lock_timeout": {
+          "name": "Lock timeout",
+          "description": "Seconds until lock mode times out. Should only be used if operation type is `timed`."
+        },
+        "outside_handles_can_open_door_configuration": {
+          "name": "Outside handles can open door configuration",
+          "description": "Which outside handles can open the door."
```

Comment:
I think we need to explain the format of the value more as it's a list of four boolean values.